torch/csrc/inductor/aoti_runtime/sycl_runtime_wrappers.h

```diff
@@ -72,7 +72,7 @@ static std::unique_ptr<sycl::kernel> _createKernel(
 
   sycl::device& syclDevice =
       c10::xpu::get_raw_device(c10::xpu::current_device());
```

Comment:
```suggestion
```

Reply:
Done in https://github.com/pytorch/pytorch/pull/154360